docs/core/macos-prerequisites.md

```diff
@@ -45,32 +24,30 @@ See [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/ma
 
 For download links and more information, see [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
 
----
+# [.NET Core 2.x](#tab/netcore2x)
 
-## .NET Core dependencies
+.NET Core 2.x is supported on the following versions of macOS:
 
-# [.NET Core 2.x](#tab/netcore2x)
+* macOS 10.12 "Sierra" and later versions
 
-Download and install the .NET Core SDK from the [.NET Downloads](https://dotnet.microsoft.com/download) page. If you have problems with the installation on macOS, consult the [Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.1) topic for the version you have installed.
+See [.NET Core 2.2 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md) and [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md) for the complete list of .NET Core 2.2 and .NET Core 2.1 supported operating systems, distributions and versions, out of support OS versions, and lifecycle policy links.
 
-# [.NET Core 1.x](#tab/netcore1x)
+For download links and more information, see [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2) or [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1).
 
-.NET Core 1.x requires OpenSSL when running on macOS. An easy way to obtain OpenSSL is by using the [Homebrew ("brew")](https://brew.sh/) package manager for macOS. After installing *brew*, install OpenSSL by executing the following commands at a Terminal (command) prompt:
+---
 
-```console
-brew update
-brew install openssl
-mkdir -p /usr/local/lib
-ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/
-ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/
-```
+## .NET Core dependencies
 
-Download and install the .NET Core SDK from the [.NET Downloads](https://dotnet.microsoft.com/download) page. If you have problems with the installation on macOS, consult the [1.0.0 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.0-known-issues.md) and [1.0.1 Known Issues](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.1-known-issues.md) topics.
+<!-- markdownlint-disable MD024 -->
 
 # [.NET Core 3.0](#tab/netcore30)
 
 Download and install the .NET Core SDK from the [.NET Downloads](https://dotnet.microsoft.com/download) page. If you have problems with the installation on macOS, consult the [Release notes](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md) topic for the version you have installed.
 
+# [.NET Core 2.x](#tab/netcore2x)
+
+Download and install the .NET Core SDK from the [.NET Downloads](https://dotnet.microsoft.com/download) page. If you have problems with the installation on macOS, consult the [Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.1) topic for the version you have installed.
```

Comment:
.NET Core 2.1 is EOL, but the known issues link goes to its known issues, not .NET Core 2.2.

Reply:
@rpetrusha Hey Ron. 2.1 is in LTS, not EOL. 2.0 is EOL though. https://github.com/dotnet/core/blob/master/microsoft-support.md

Yes?

Reply:
It's still a better experience to link to a specific .NET Core 2.x download page instead of <https://dotnet.microsoft.com/download>, which is a .NET Core 3.0 page with an option to go to a page with other versions. And if you're covering both 2.1 and 2.2, there should be two separate known issues links, one for each version.